### ISSUE TITLE
Set the range of value in Single Input of type range

### ISSUE THREAD

**Issue by GuiAfonso** [surveyjs/surveyjs#444]
### Are you requesting a feature, reporting a bug or ask a question?
I am asking a question

### What is the current behavior?
Editing the question I can't find a place to set the range of values to a range type input. And I can't find an example too.

### What is the expected behavior?
I want to know where I put the range of number in my input with type range.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
your_code_here

```

### Specify your

* browser: Chrome
* browser version: 58.0.3029.110
* surveyjs platform (angular or react or jquery or knockout or vue): jquey
* surveyjs version: 0.12.16

**Comment by GuiAfonso** [comment]
@andrewtelnov can you help me in this silly question too? Thanks

**Comment by andrewtelnov** [comment]
@GuiAfonso Sure, use the rateValues property:
http://surveyjs.org/builder/#48323384-676a-46b6-b4be-8c10c8b5c0fb

**Comment by GuiAfonso** [comment]
Thanks again @andrewtelnov, I'am already using rateValues in other question. Single input of type range don't have a rateValues tab. 

[http://surveyjs.org/builder/#48323384-676a-46b6-b4be-8c10c8b5c0fb](url)
I add a question in the above link. Can you see please?

**Comment by andrewtelnov** [comment]
@GuiAfonso Sorry, I do not get you. Are you talking about dropdown question?
http://surveyjs.org/examples/jquery/questiontype-dropdown/

**Comment by GuiAfonso** [comment]
@andrewtelnov If I add a simple input, and change that input to range type (it will change the input to a slider) how i add the range of that slider(the interval of values). In this case, I don't have a rateValues tab.

**Comment by andrewtelnov** [comment]
@GuiAfonso Yes, I got you finally. Sorry about that.
We will add into text 3 properties: rangeMin, rangeMax and rangeStep.
Here is their description: https://www.w3schools.com/jsref/dom_obj_range.asp

Thank you,
Andrew

**Comment by JayDLX** [comment]
Hey there.

How is the state about adding this feature ? 
I still cannot find or see it ... or did you add it somewhere else?

Also it would be nice, if the user can see directly, which value he is currently sliding and what are the Minimum, Maximum.

Best regards,
Julian

**Comment by andrewtelnov** [comment]
@JayDLX Sorry, that I did write back to you. After the discussion, we decided that it is probably better introduce a new range question. The only problem we have with it, that it will look and have different behavior in different browsers. As result we did not come to any solution.
Will the custom solution work for you? 
https://www.surveyjs.io/Examples/Library/?id=custom-widget-nouislider

We may update the demo and add minimum, maximum and some other properties.

Thank you,
Andrew
